packages/components/src/components/expandable/expandable.stories.ts

```diff
@@ -30,222 +33,52 @@ export const Default = {
- */
-export const Inverted = {
-  parameters: { controls: { exclude: 'inverted' } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'inverted' }
-      },
-      args,
-      options: {
-        templateBackgrounds: { alternate: 'y', colors: ['rgb(var(--sd-color-primary, 0 53 142))', 'white'] }
-      }
-    });
-  }
-};
-
-/**
- * Use the `default`, `toggle-open` and `toggle-closed` slots to add content to the expandable.
- */
-export const Slots = {
-  parameters: {
-    controls: { exclude: ['default', 'toggle-open', 'toggle-closed'] }
-  },
-  render: (args: any) => {
-    return html`
-      ${['default', 'toggle-open', 'toggle-closed'].map(slot =>
-        generateTemplate({
-          axis: {
-            x: {
-              type: 'slot',
-              name: slot,
-              title: 'slot=...',
-              values: [
-                {
-                  value:
-                    slot === 'default'
-                      ? `<div class="slot slot--border slot--background slot--text h-full">Default slot</div>`
-                      : `<div slot='${slot}' class="slot slot--border slot--background slot--text h-12"></div>`,
-                  title: slot
-                }
-              ]
-            }
-          },
-          args
-        })
-      )}
-    `;
-  }
-};
-
-/**
- * Use the `content`, `toggle`, `summary` and `details` parts to style the expandable.
- */
-export const Parts = {
-  parameters: {
-    controls: {
-      exclude: ['open', 'content', 'toggle', 'summary', 'details']
-    }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'template',
-          name: 'sd-expandable::part(...){outline: solid 2px red}',
-          values: ['content', 'toggle', 'summary', 'details'].map(part => {
-            return {
-              title: part,
-              value: `<style>#part-${part} sd-expandable::part(${part}){outline: solid 2px red; outline-offset: -2px;}</style><div id="part-${part}">%TEMPLATE%</div>`
-            };
-          })
-        }
-      },
-      args
-    });
-  }
-};
-
-/**
- * sd-expandable is fully accessibile via keyboard.
+ * Use the `open` attribute to set the state of the expandable.
  */
 
-export const Mouseless = {
-  render: (args: any) => {
-    return html`<div class="mouseless">${generateTemplate({ args })}</div>`;
-  },
-
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-expandable');
-    await waitUntil(() => el?.shadowRoot?.querySelector('button'));
-    await userEvent.type(el!.shadowRoot!.querySelector('button')!, '{return}', { pointerEventsCheck: 0 });
+export const Open = {
+  name: 'Open',
+  render: () => {
+    return html`<div class="flex flex-col gap-12">
+      <sd-expandable>
+        <div class="sd-paragraph">
```

Comment:
Why is this a `div`? Please make it a `p` and use `sd-paragraph` here or even better `sd-prose` on the `sd-expandable`